LEAF_Request_Portal/sources/Group.php

```diff
@@ -269,54 +269,82 @@ public function getMembers($groupID, bool $searchDeleted = false): array|string
      *
      * @return string
      */
-    public function addMember($member, $groupID): string
+    public function addMember($member, $groupID): array
```

Comment:
Do we want to do type-hinting here?  Or might that cause issues?  I ask because normally you're pretty good about those and if you didn't use them here, I'm guessing there's a reason.

Reply:
I hadn't thought about it and just missed it in this case. The string hint was already there and I needed to change that obviously but didn't even cross my mind for the others. Not sure if we might run into anything if we add them here. I did look to see where this method is used and decided to add them. Thing is if the come in as null (which I think is the biggest culprit here) and it adds it to the db it would be a dead record with no way to get the data because the userID and groupID are required for retrieval.